api-report/datastore.api.md

```diff
@@ -89,6 +88,7 @@ export class FluidDataStoreRuntime extends TypedEventEmitter<IFluidDataStoreRunt
     static load(context: IFluidDataStoreContext, sharedObjectRegistry: ISharedObjectRegistry, existing: boolean): FluidDataStoreRuntime;
     // (undocumented)
     readonly logger: ITelemetryLogger;
+    makeVisibleAndAttachGraph(): void;
```

Comment:
this looks like a breaking change. does this need to be made in the next branch

Reply:
or should this be and optional method?

Reply:
It is optional on IFluidDataStoreChannel. This here is the class implementation.